mount/mount_linux.go

```diff
@@ -169,31 +183,39 @@ func detectSystemd() bool {
 }
 
 // MakeMountArgs makes the arguments to the mount(8) command.
-// Implementation is shared with NsEnterMounter
-func MakeMountArgs(source, target, fstype string, options []string) []string {
```

Comment:
taking a public method private is not compatible

Reply:
How strict do we want to be about that? As far as I can tell they are internal implementation details. I'd love to reduce the API surface if we can. The most important consumer of this is Kubernetes and it does not reference these methods.

Reply:
> How strict do we want to be about that?

As strict as we can? The intent of this library is to be a well-behaved, stable dependency. I agree with limiting API surface area, but the time to do that is before we publish releases.

Reply:
Ack. Will reintroduce and add a sensitive version as well.